static/docs/commands-reference/import.md

```diff
@@ -21,28 +12,278 @@ Supported schemes:
+
+When DVC inspects a DVC file, one step is inspecting the dependencies to see if
+any have changed. A changed dependency will appear in the `dvc status` report,
+indicating the need to re-run the corresponding part of the pipeline. When DVC
+inspects an external dependency, it uses a method appropriate to that dependency
+to test its current status.
+
 ## Options
 
+* `--resume` - resume previously started download.
+
+* `-f`, `--file` - specify name of the DVC file it generates. It should be 
+  either `Dvcfile` or have a `.dvc` suffix (e.g. `data.dvc`) in order 
+  for `dvc` to be able to find it later. 
+
 * `-h`, `--help` - prints the usage/help message, and exit.
 
 * `-q`, `--quiet` - does not write anything to standard output. Exit with 0 if
   no problems arise, otherwise 1.
 
 * `-v`, `--verbose` - displays detailed tracing information.
 
-* `--resume` - resume previously started download.
+## Example: Initializing a workspace using a remote file
 
-* `-f`, `--file` - specify name of the DVC file it generates. It should be 
-  either `Dvcfile` or have a `.dvc` suffix (e.g. `data.dvc`) in order 
-  for `dvc` to be able to find it later. 
+The [DVC getting started tutorial](/doc/get-started) demonstrates a simple DVC
+pipeline.  In the [Add Files step](/doc/get-started/add-files) we are told to
+download a file, then use `dvc add` to integrate it with the workspace.
+
+An alternate way to initialize the _Getting Started_ workspace, using
+`dvc import`, is
+
+```dvc
+    $ mkdir get-started
+    $ cd get-started
+    $ git init
+    $ dvc init
+    $ mkdir data
+    $ dvc import https://dvc.org/s3/get-started/data.xml data/data.xml
+    Importing 'https://dvc.org/s3/get-started/data.xml' -> '/Volumes/Extra/dvc/get-started/data/data.xml'
+    [##############################] 100% data.xml
+    Adding 'data/data.xml' to 'data/.gitignore'.
+    Saving 'data/data.xml' to cache '.dvc/cache'.
+    Saving information to 'data.xml.dvc'.
+
+    To track the changes with git run:
+
+        git add data/.gitignore data.xml.dvc
+```
+
+If you wish, it's possible to set up the other stages from the _Getting Started_
+example. Since we do not need those stages for this example, we'll skip that.
+Instead we can look at the resulting DVC file `data.xml.dvc`:
+
+```yaml
+    deps:
+    - etag: '"f432e270cd634c51296ecd2bc2f5e752-5"'
+      path: https://dvc.org/s3/get-started/data.xml
+    md5: 61e80c38c1ce04ed2e11e331258e6d0d
+    outs:
+    - cache: true
+      md5: a304afb96060aad90176268345e10355
+      metric: false
+      path: data/data.xml
+      persist: false
+    wdir: .
+```
+
+The `etag` field in the DVC file contains the ETAG recorded from the HTTP
+request. If the remote file changes, the ETAG changes, letting DVC know when
+the file has changed. 
+
+## Example: Remote file that is updated
 
-## Examples
+What if that remote file is one which will be updated regularly? The project
```

Comment:
Thes idea is great! But let's simplify it - let's use the local "remote" path as an example instead of SSH. The same `data.xml`, and we can skip some outputs in this case then. It looks too big and hard to reproduce now.